tests/Resources/Page/ODH/ODHDashboard/ODHDashboardResources.resource

```diff
@@ -0,0 +1,90 @@
+*** Settings ***
+Resource      ../../../Page/Components/Components.resource
+Resource       ../../../Common.robot
+
+
+*** Keywords ***
+Open QuickStart Element in Resource Section By Name
+    [Documentation]     It opens the element by enabling Quick Start filter in Resources Section of ODS Dashboard.
+    [Arguments]     ${element}
+    Select Checkbox     //input[@id="quickstart--check-box"]
+    Wait Until Page Contains Element    //article[@id="${element}"]
+    Click Link          //article[@id="${element}"]//a
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+
+Star QuickStart Tour
+    Click Button    //button[@data-test="Start button"]
+    Wait Until Page Contains Element    //div[@class="pfext-quick-start-content"]
+
+Get The Count Of QuickStart Steps
+    [Documentation]     Gives the count of steps in a the quick start
+    ${count}=   Get ELement Count   //div[@class="pfext-quick-start-content"]/ul/li
+    [Return]    ${count}
+
+Link Text On QuickStart Card Should Be
+    [Arguments]     ${element}  ${exp_link_text}
+    Element Text Should Be  //article[@id="${element}"]//a      ${exp_link_text}
+
+QuickStart Status Should Be
+    [Documentation]     Verifies the label on the item card which shows the progress and closes the sidebar.
+    [Arguments]     ${element}  ${progress_string}
+    IF  '${progress_string}'=='In Progress'
+        Element Text Should Be      //article[@id="${element}"]//span[@class="pf-c-label pf-m-purple pf-m-outline"]     In Progress
+    ELSE IF     '${progress_string}'=='Complete'
+        Element Text Should Be      //article[@id="${element}"]//span[@class="pf-c-label pf-m-green pf-m-outline"]      Complete
+    ELSE IF     '${progress_string}'=='Failed'
+        Element Text Should Be      //article[@id="${element}"]//span[@class="pf-c-label pf-m-red pf-m-outline"]        Failed
+    END
+
+QuickStart Status Should Not Be Set
```

Comment:
A lot of [Documentation] tags missing in this file. If you think these keywords don't need it because they are too simple you can disable the rule with
`Some Keyword  # robocop: disable:missing-doc-keyword`

Reply:
nono, they need documentation and this will be added in future PR soon after this one